### PR TITLE
Add Manila 1.0.1

### PR DESCRIPTION
**Pull request by danielbayley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/manila.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  desc "Finder extension for changing folder colors"
+  homepage "https://github.com/neilsardesai/Manila#readme"
+
+  livecheck do
+    url :stable
+    strategy :git do |tags|
+      tags.delete "v1.0.2"
+      tags.map { |tag| tag.delete_prefix "v" }
+    end
+  end
```
Comment (carlocab) [inline Casks/manila.rb:16]: This seems like a strange `livecheck` block. Is there anything particular about the `v1.0.2` tag that merits excluding it, other than that it's not registered as a release on GitHub? If so, why not just use `:github_latest`?

Comment (danielbayley) [reply Casks/manila.rb]: Ah I thought it was picking up all the tags, even though that particular tag is currently missing the asset… Fixed.

**Review by carlocab — COMMENTED** [review]
```
audit for manila: failed
 - GitHub repository too new (<30 days old)
Error: 1 problem in 1 cask detected
Error: GitHub repository too new (<30 days old)
```
Let's wait until the GitHub repository is old enough.

File: Casks/manila.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  url "https://github.com/neilsardesai/Manila/releases/download/v#{version}/Manila.zip"
+  name "Manila"
+  desc "Finder extension for changing folder colors"
+  homepage "https://github.com/neilsardesai/Manila#readme"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
```
Comment (bevanjkay) [inline Casks/manila.rb:14]: @danielbayley The livecheck block can be ommitted here as the default `:git` strategy will be inferred automatically for the URL.

Comment (danielbayley) [reply Casks/manila.rb]: @bevanjkay But then we get this: https://github.com/Homebrew/homebrew-cask/runs/6065942618?check_suite_focus=true#step:11:13

Comment (bevanjkay) [reply Casks/manila.rb]: @danielbayley - in that case the `:github_latest` livecheck block will need to be included. When I checked it previously there was no tag that was newer than the latest release. Thanks

**Review by miccal — APPROVED** [review]

**Pull request merged**